analyzers/src/SonarAnalyzer.CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -363,17 +363,6 @@ static bool TakesExpressionTree(SymbolInfo info)
             }
         }
 
-        public static bool IsParentKind<T>(this SyntaxNode node, SyntaxKind kind, out T result) where T : SyntaxNode
```

Comment:
To be consistent, I moved some of these pre-existing extension methods to the "Roslyn" file.